### ISSUE TITLE
Xamarin/PCL Support?

### ISSUE THREAD

**Issue by ghost** [vtortola/WebSocketListener#32]
Greetings! This looks like a fantastic API. Is there a chance it can be modified slightly to be PCL-compliant and thus support the ever-growing Xamarin user base? I'd be happy to contribute beer money :)

I saw a closed issue from another user stating that they had to modify the: "default buffer, not buffer manager, and remove some of the .Net logging statements to get it to compile for PCL IOS and Android". Thanks for reading. Cheers from Florida!

**Comment by vtortola** [comment]
Hi! I have never used Xamarin and to be honest it is not in my TODO list. But I am happy to discuss changes :)

**Comment by jasonCodesAway** [comment]
On further research looks like it may have been done already! :) 
https://github.com/rdavisau/sockets-for-pcl

**Comment by vtortola** [comment]
Nice :)

**Comment by pushplay** [comment]
I've successfully created a nuget package that supports Xamarin iOS and Android with no additional dependencies.  I did it by creating csprojs for each platform.  My license doesn't cover Xamarin Mac but I'm sure it would work just the same.

I don't know if you'd want me to contribute this or how that would look.  Having one csproj per project, per platform is a maintenance headache.  The project.json project format is going to allow multiple runtime targets but for now we're still in the cross-platform dark ages.
